src/hotspot/share/gc/shared/vmStructs_gc.hpp

```diff
@@ -175,9 +170,6 @@
   declare_toplevel_type(CardTableBarrierSet**)                            \
   declare_toplevel_type(CollectedHeap*)                                   \
   declare_toplevel_type(ContiguousSpace*)                                 \
-  declare_toplevel_type(DefNewGeneration*)                                \
-  declare_toplevel_type(GenCollectedHeap*)                                \
-  declare_toplevel_type(Generation*)                                      \
```

Comment:
You need to restore these lines and rename GenCollectedHeap -> SerialHeap. I'm not sure why DefNewGeneration and Generation were deleted. I assume they are still present.

Reply:
It looks like all 3 of these already exist elsewhere, thus their removal is ok.